pkgs/tools/system/ddrescueview/default.nix

```diff
@@ -1,20 +1,24 @@
 { stdenv, lib, fetchurl, fpc, lazarus, atk, cairo, gdk-pixbuf, glib, gtk2, libX11, pango }:
 
-stdenv.mkDerivation rec {
-  name = "ddrescueview-0.4alpha3";
+let
+  versionBase = "0.4";
+  versionSuffix = "alpha4";
+in stdenv.mkDerivation rec {
+  pname = "ddrescueview";
```

Comment:
Wrong branch?

Reply:
No. As I said in the PR description, `ddrescueview` would be broken by the fpc update and the update fixes it. The `ultrastardx` update is not strictly necessary (the patch would also apply to the old version), but why not do it while I'm at it.

Reply:
Ah, a race condition in our GitHub use, I guess.